src/dune_engine/diff_promotion.ml

```diff
@@ -179,3 +178,26 @@ let promote_files_registered_in_last_run files_to_promote =
   let db = load_db () in
   let db = do_promote db files_to_promote in
   dump_db db
+
+let diff_for_file (file : File.t) =
+  let msg = User_message.Annots.empty in
+  let a = Path.source file.dst in
+  let b = File.correction_file file in
```

Comment:
We should pick better names like `in_source`, and `in_build` or just inline these variables.

Reply:
The `a`, `b` names were in reference to `a/` and `b/` paths in diffs. But you're right, it's better to be explicit.